frontend/src/components/pages/camps/CamperDetailsBadges.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+
+import {
+  Container,
+  Text,
+  Image,
+  HStack,
+} from "@chakra-ui/react";
+
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faHandDots,
+  faHandshakeAngle,
+} from "@fortawesome/free-solid-svg-icons";
+
+import icon_pickup from "../../../assets/icon_custom_sunrise.svg";
+
+import { Camper } from "../../../types/CamperTypes";
+
+export const CamperDetailsBadge = ({
+    customIcon,
+    icon,
+    description,
+    color,
+  }: {
+    customIcon: boolean;
+    icon: any;
+    description: string;
+    color: string;
+  }) => {
+    return (
+      <HStack
+        alignContent="center"
+        background={color}
+        px="5"
+        py="2"
+        borderRadius="5"
+      >
+        {customIcon ? (
+          <Image src={icon} display="inline" maxWidth="22px" />
+        ) : (
+          <FontAwesomeIcon icon={icon} />
+        )}
+        <Text>{description}</Text>
+      </HStack>
+    );
+  };
+  
+  export const CamperDetailsBadgeGroup = ({ camper }: { camper: Camper }) => {
+    return (
+      <Container width="-webkit-fit-content" marginStart="0px">
+        <HStack>
+          {camper.latePickup && camper.earlyDropoff && (
```

Comment:
I think there are 3 states for EDLP:
1. Late pickup AND NOT Early Dropoff
2. Early Dropoff AND NOT Late pickup
3. BOTH Late pickup and early dropoff 

<img width="881" alt="image" src="https://user-images.githubusercontent.com/57159970/186121103-241a4832-0c2d-4921-8aba-ad243c7b8ce6.png">

Reply:
I think we might just remove the third status though so just hold off on this change for now